src/index.html

```diff
@@ -9,6 +9,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img alt="logo"
+      src="images/logo.png"
+      class="logo-icon"
+      >
+      </a>
```

Comment:
Keep correct indentations for the elements.
```suggestion
      <a href="#">
        <img
          alt="logo"
          src="images/logo.png"
          class="logo-icon"
        >
      </a>
```